src/parser.ml

```diff
@@ -287,9 +287,13 @@ let parser term @(p : prio) =
   (* Products. *)
   | "∀" xs:arg+ "," b:(term PFunc)
       when p >= PFunc -> in_pos _loc (P_Prod(xs,b))
+  | "∀" xs:arg_ident+ ":" a:(term PFunc) "," b:(term PFunc)
+     when p >= PFunc -> in_pos _loc (P_Prod([xs,Some(a),false],b))
```

Comment:
The indentation of the `when` is not correct, but otherwise I can't see any problem.

Reply:
Fixed in 6dfeeee506300c83218b2f1d245b99158a08be72